src/content/docs/dev/send-tokens/interchain-tokens/developer-guides/link-custom-tokens-deployed-across-multiple-chains-into-interchain-tokens.mdx

```diff
@@ -433,53 +478,112 @@ async function main() {
 
-<Callout emoji="💡">
-When deploying the token manager to a preferred chain other than the local chain (in our example, the Fantom testnet) while the remote chain is Polygon, make sure to deploy the remote token manager from the local chain. If you deploy to the Polygon testnet, you may encounter a `TokenManagerDoesNotExist` error. This error occurs because there is no token manager on Polygon with the `tokenId` you are using. You don’t need to deploy a remote token manager for Polygon because there is already one available on Fantom. Therefore, we deploy that token manager from the Fantom testnet.
-
-</Callout>
+```
 
 ### Check the transaction on the Axelar testnet scanner
 
-Check the [Axelarscan testnet scanner](https://testnet.axelarscan.io/) to see if you have successfully deployed the remote token manager on the Polygon testnet. It should look something like [this](https://testnet.axelarscan.io/gmp/0x5dbba37f7741d19ca872bb4b2a29523baa002a3d6f0f31834bc34d6d6ca633f5).
+Check the [Axelarscan testnet scanner](https://testnet.axelarscan.io/) to see if you have successfully deployed the remote token manager on the Polygon testnet. It should look something like [this](https://testnet.axelarscan.io/gmp/0x11c2a212180d7ea2babe7aceb66c32c1e235cc159bceeb4dd6d04411f4a9669f).
 
 Add gas if needed. Ensure that Axelar shows a successful transaction before continuing to the next step.
 
-## Transfer mint access to the Token Manager address on the Fantom and Polygon testnets
+## Assign the minter role to your token's Token Manager on BSC and Avalanche Fuji chain
 
 You must transfer mint access to the Token Manager address on both chains before you can mint and burn tokens while moving assets between chains.
 
-### Transfer mint access to the Token Manager address on the Fantom testnet
+### Transfer mint access to the Token Manager address on the BSC testnet
+
+Create a `transferMintAccessToTokenManagerOnBSC()` function that will perform the mint access transfer on the BSC testnet. The token manager is a parameter needed for granting mint access. You need to first retrieve the token manager address using the token ID. So, let's start with that. To do that, use the following code snippet.
+
+```javascript
+async function getTokenManagerAddress() {
+  // Get a signer to sign the transaction
+  const signer = await getSigner();
+
+  // Get the InterchainTokenService contract instance
+  const interchainTokenServiceContract = await getContractInstance(
+    interchainTokenServiceContractAddress,
+    interchainTokenServiceContractABI,
+    signer,
+  );
+
+  // Register token metadata
+  const tokenId = await interchainTokenServiceContract.interchainTokenId(
+    "0x510e5EA32386B7C48C4DEEAC80e86859b5e2416C", // sender
+    "0xd58c9142d863dde9f0b9ea68b10cce9470626040187a00c16b7b3d4edae70cf9", // salt, same as previously used
+  );
+
+  const tokenManagerAddress =
+    await interchainTokenServiceContract.tokenManagerAddress(tokenId);
```

Comment:
I think we should query `deployedTokenManager()` instead it will ensure that the manager actually exists rather than just running the create3 address on the `tokenId`

Reply:
But it's the same, isn't it? It is still being retrieved by the token ID. I think it doesn't matter what it's called; the important thing is to retrieve it.

Reply:
It does return the same address. In practice I think the other function is safer since it will revert if the manager does not exist which is important for people to know.